elasticdl/python/common/tensor.py

```diff
@@ -86,23 +130,29 @@ def set(self, values=None, indices=None, name=None):
                     "When creating a Tensor object with values of type "
                     "tf.IndexedSlices, indices must be None."
                 )
+            if values.dense_shape is not None:
+                # TODO(yunjian.lmh): Support dense shape, or do not print
+                #     warning message, or there will be too much warning
+                #     messages.
+                logger.warning(
+                    "Convert a TensorFlow.IndexedSlices object with dense "
+                    "shape to ElasticDL Tensor. ElasticDL ignores dense shape "
+                    "now."
```

Comment:
"ElasticDL Tensor ignores dense_shape in TensorFlow.IndexedSlices."

Reply:
Done.